docs/Onboarding Docs/01_Razor_Basics.md

```diff
@@ -0,0 +1,52 @@
+﻿# Razor Basics
+
+Razor is a templating language used in ASP.NET for creating dynamic web pages. It's not a programming language itself, but a server-side markup syntax for embedding server-side code (C# or VB.NET) into HTML.
+
+In a Razor file, you can use a combination of several languages:
+
+| Language | Usage | Supported in .NET Core and .NET 5+ |
+| --- | --- | --- |
+| **Razor syntax** | Used to embed and execute server-side code within HTML. | Yes |
+| **C#** | The server-side language used within Razor templates. Most commonly used with Razor. | Yes |
+| **HTML** | Used to structure the content on web pages. | Yes |
+| **JavaScript** | Used for client-side scripting in Razor templates. | Yes |
+| **CSS** | Used for styling web pages. | Yes |
+| **VB.NET** | Can be used in Razor syntax in the older .NET Framework. | No |
+
+Please note that while Razor syntax does support VB.NET in the older .NET Framework, VB.NET is not supported in .NET Core or .NET 5 and onwards for Razor views. In these newer frameworks, only C# is supported for Razor views.
```

Comment:
```suggestion
Please note that while Razor syntax does support VB.NET in the older .NET Framework, VB.NET is not supported in .NET Core or .NET 5 and onwards. In these newer frameworks, only C# is supported.
```

Calling out "for Razor views" here makes it seem like it is supported in the other file types listed below, which it is not.